lib/src/dynamic_color_plugin.dart

```diff
@@ -32,7 +33,16 @@ class DynamicColorPlugin {
     return result == null ? null : CorePalette.fromList(result.toList());
   }
 
-  /// Returns the macOS' control accent color asynchronously in an [int] color.
-  static Future<int> getControlAccentColor() async =>
-      await channel.invokeMethod(controlAccentColorMethodName);
+  /// Returns the macOS' control accent color asynchronously in a [Color].
+  ///
+  /// Supported since macOS 10.14, Mojave
```

Comment:
```suggestion
  /// Supported since macOS 10.14 (Mojave).
  ///
  /// Completes will null on older versions of macOS 10.14, and on non-macOS platforms.
```

Reply:
Done